src/LiveSplit.Delta/UI/Components/DeltaComponent.cs

```diff
@@ -156,6 +148,19 @@ public void Update(IInvalidator invalidator, LiveSplitState state, float width,
                 InternalComponent.TimeValue = null;
             }
 
+            var text = comparisonName;
+            if (Settings.OverrideText)
+            {
+                text = Settings.DifferentialText && InternalComponent.TimeValue < TimeSpan.Zero ? Settings.TextAhead : Settings.TextBehind;
+            }
+            if (InternalComponent.InformationName != text)
+            {
+                InternalComponent.AlternateNameText.Clear();
+                InternalComponent.AlternateNameText.Add(CompositeComparisons.GetShortComparisonName(comparison));
```

Comment:
Are you sure you want this? This will show the short comparison name instead of the custom text.

Reply:
If I'm understanding correctly, it will only display the short comparison name if the custom text is deemed too long. Though when thinking about it again; it seems counter-intuitive enough to change this behaviour.